src/core/thread/mle.cpp

```diff
@@ -2315,14 +2363,19 @@ ThreadError Mle::HandleAnnounce(const Message &aMessage, const Ip6::MessageInfo
 
     localTimestamp = mNetif.GetActiveDataset().GetNetwork().GetTimestamp();
```

Comment:
here, do we need to call `mNetif.GetActiveDataset().GetLocal().GetTimestamp()` to obtain local timestamp? in my understanding, `GetNetwork()` will return the dataset disseminated in network(i.e. ChildIdResponse, DataResponse).